site/docs/master/annotations.md

```diff
@@ -13,15 +13,37 @@ However, Contour still supports a number of annotations on the Ingress resources
 
 The following Kubernetes annotations are supported on [`Ingress`] objects:
```

Comment:
Why are there square brackets here?

Reply:
This one was inherited, no idea, I'll remove.